torch_geometric/nn/aggr/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .aggr import Aggr, MeanAggr
+
+__all__ = classes = [
+    'Aggr',
+    'MeanAggr',
+]
```

Comment:
good place for class-resolver?

Reply:
Thanks for your input. We will add that in a follow-up PR :)

Reply:
I will be happy to take care of it! I'll get a notification when this one is merged and follow up ASAP

Reply:
See https://github.com/pyg-team/pytorch_geometric/pull/4716